js/cryptomarket.js

```diff
@@ -0,0 +1,736 @@
+            // };
+            result.push ({
+                'id': currency_pair,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'info': currency_pair,
+            });
+        }
+        return result;
+    }
+
+    parseOrderStatus (status) {
+        const statuses = {
+            'queued': 'open',
+            'filled': 'closed',
+            'cancelled': 'canceled', // turns out both spellings are correct, the first its preferred by british writers, and the second by north american writers.
+        };
+        return this.safeString (statuses, status, status);
+    }
+
+    parseOrder (order, market = undefined) {
+        // if is a closed order
+        // {'amount': {'executed': '1', 'original': '1'},
+        //    'avg_execution_price': '47.85',
+        //    'created_at': '2020-02-26T20:56:42.360000',
+        //    'fee': '0.325',
+        //    'fills': [{'amount': '47.85',
+        //               'date': '2020-02-26T20:56:42',
+        //               'fee': '0.325',
+        //               'price': '47.85'}],
+        //    'id': 'O614553',
+        //    'market': 'XLMCLP',
+        //    'price': '0', // 0 if a market order, else it has value
+        //    'side': 'sell',
+        //    'status': 'filled',
+        //    'stop': None,
+        //    'type': 'market',
+        //    'updated_at': '2020-02-26T20:56:43.460454'
+        // }
+        // if is an open order, then is a subset of a closed order (no fills and fee = 0)
+        // {
+        //     'amount': {'executed': '0', 'original': '1'},
+        //     'avg_execution_price': '0',
+        //     'created_at': '2020-05-18T01:49:33.302000',
+        //     'fee': '0',
+        //     'id': 'O9761171',
+        //     'market': 'XLMCLP',
+        //     'price': '120',
+        //     'side': 'sell',
+        //     'status': 'queued',
+        //     'stop': None,
+        //     'type': 'limit',
+        //     'updated_at': '2020-05-18T01:49:33.342640'
+        // }
+        const id = this.safeString (order, 'id');
+        const status = this.parseOrderStatus (this.safeString (order, 'status'));
+        const amountDict = this.safeValue (order, 'amount');
+        const amount = this.safeFloat (amountDict, 'original');
+        const filled = this.safeFloat (amountDict, 'executed');
+        const remaining = amount - filled;
+        const price = this.safeFloat (order, 'avg_execution_price');
+        const type = this.safeString (order, 'type');
+        const side = this.safeString (order, 'side');
+        const fee = this.safeFloat (order, 'fee');
+        const created_at = this.safeString (order, 'created_at');
+        //
+        const fills = this.safeValue (order, 'fills');
+        let lastTradeTimestamp = undefined;
+        let cost = 0.0;
+        if (fills !== undefined) {
```

Comment:
We will need to move this fill-parsing section and pass it through `parseTrades() / parseTrade()` callchain, populating the `trades` field below in line 270 https://github.com/ccxt/ccxt/pull/7058/files#diff-4c2a5ea9da37b6c46c2936642cff7005R270

Reply:
done, the new version is up